File: TradingEAC-FrontEnd/src/app/models/company-profile.ts
export class CompanyProfile {
    constructor(){
        this.symbol = '';
        this.country = '';
        this.currency = '';
        this.exchange = '';
        this.name = '';
        this.ticker = '';
        this.ipo = new Date();
        this.marketCapitalization = 0;
        this.shareOutstanding = 0;
        this.logo = '';
        this.phone = '';
        this.weburl = '';
        this.finnhubIndustry = '';
        this.currentPrice = 0;
        this.previousPrice = 0;
    }
    public symbol : string;
    public country : string;
    public currency : string;
    public exchange : string;
    public name : string;
    public ticker : string;
    public ipo : Date;
    public marketCapitalization : number;
    public shareOutstanding : number;
    public logo : string;
    public phone : string;
    public weburl : string;
    public finnhubIndustry : string;
    public currentPrice : number;
    public previousPrice : number;
}

File: TradingEAC-Java-StockService/src/main/java/tradingeac/java/stockservice/models/CompanyProfile.java
package tradingeac.java.stockservice.models;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

public class CompanyProfile {
    private String symbol;
    public void setSymbol(String value){
        symbol = value;
    }
    public String getSymbol(){
        return symbol;
    }
    
    @Getter @Setter private String country;
    @Getter @Setter private String currency;
    @Getter @Setter private String exchange;
    @Getter @Setter private String name;
    @Getter @Setter private String ticker;
    @Getter @Setter private Date ipo;
    @Getter @Setter private double marketCapitalization;
    @Getter @Setter private double shareOutstanding;
    @Getter @Setter private String logo;
    @Getter @Setter private String phone;
    @Getter @Setter private String weburl;
    @Getter @Setter private String finnhubIndustry;
}
